llvm/test/tools/llvm-cvtres/help.test

```diff
@@ -1,16 +1,16 @@
 ; RUN: llvm-cvtres /h > %t
-; RUN: FileCheck -input-file=%t %s -check-prefix=HELP_TEST
+; RUN: FileCheck -input-file=%t %s --strict-whitespace -check-prefix=HELP_TEST
 
-; HELP_TEST: 	  OVERVIEW: Resource Converter
-; HELP_TEST-DAG:  USAGE: llvm-cvtres [options] file...
-; HELP_TEST-DAG:  OPTIONS:
-; HELP_TEST-NEXT:   /{{DEFINE}}:symbol - Not implemented
-; HELP_TEST-NEXT:   /FOLDDUPS: - Not implemented
-; HELP_TEST-NEXT:   /HELP - Display available options
+;      HELP_TEST: OVERVIEW: Resource Converter
+;  HELP_TEST-DAG: USAGE: llvm-cvtres [options] file...
+;  HELP_TEST-DAG: OPTIONS:
+; HELP_TEST-NEXT:   /{{DEFINE}}:symbol     - Not implemented
```

Comment:
The formatting in this line looks wrong? It looks to me like there might be a bug here (possibly predating your PR).

Reply:
This is a quirk of FileCheck. It thinks that `DEFINE` is a directive, so in order to match the `DEFINE` word it is necessary to make it a regex. FileCheck will ignore `{{}}` and that is why it doesn't look at the same indentation level as the rest of the options, but it is.